src/PrestaShopBundle/Controller/Admin/Improve/International/CurrencyController.php

```diff
@@ -429,6 +494,10 @@ private function getErrorMessages(Exception $e)
                 'You cannot disable the default currency',
                 'Admin.International.Notification'
             ),
+            InvalidUnofficialCurrencyException::class => $this->trans(
+                'You cannot create an unofficial currency whose ISO code matches a real one',
```

Comment:
As seen with @MatShir and @jolelievre (?), we opted for a more didactic notification, supposed to be dynamic as well:

_Oops... it looks like this ISO code already exists. If you are:
- trying to create a new currency, you must type a different ISO code
- trying to modify the %currency% currency, just go back to the list and edit it_

Reply:
I agree with this didactic approach, however the second message is not completely correct.
This error is thrown when you try to create an unofficial currency using a real ISO code, which is forbidden but it doesn't mean the real currency has already been installed
So it feels weird to advise to go edit a currency which won't be in the list

Reply:
Oops... it looks like this ISO code already exists. If you are:

- trying to create an alternative currency, you must type a different ISO code
- trying to modify the %currency% currency, make sure you did not check the creation box